modules/addBook.js

```diff
@@ -0,0 +1,11 @@
+const saveBooksToStorage = (bookDetails) => {
+  localStorage.setItem('booksData', JSON.stringify(bookDetails));
+};
+
+const addBook = (bookDetails, title, author) => {
+  const book = { title, author };
+  bookDetails.push(book);
+  saveBooksToStorage(bookDetails);
+};
```

Comment:
- [ ] The purpose of this milestone is to get familiar with ES6 features, for the sake of practice the more modules you define the better. However, you have to adhere to Javascript best practices as well. For example, in this case, you have defined `addBook` as a method in your class and here you created a module that **add a book**. They are the same code, which makes them not DRY 👚 👖 

So to fix that, you can take out the method `addBook` from your class and only use the method defined in your module and use the feature `import` when you want to use it.
The same goes with your `removeBook` and `updateTime` method 😁